dns/records/workspace-variables/pubtest_staging.tfvars.json

```diff
@@ -0,0 +1,30 @@
+{
+  "hosted_zone": {
+    "pubtest-testing.service123.gov.uk": {
+      "cnames": {
+        "staging": {
+          "target": "d3kffbwt0testb.cloudfront.net",
+          "ttl": 300
+        },
+        "staging-val": {
+          "target": "d3kffbwt0testc.cloudfront.net",
+          "ttl": 300
+        }
+      },
+    "resource_group_name": "s121d01-dns"
+    },
+    "pubtest-testing.service456.gov.uk": {
+      "cnames": {
+        "staging": {
+          "target": "d3kffbwt0testb.cloudfront.net",
+          "ttl": 300
+        },
+        "staging-val": {
+          "target": "d3kffbwt0testc.cloudfront.net",
+          "ttl": 300
+        }
+      },
+    "resource_group_name": "s121d01-dns"
+    }
+  }
+}
```

Comment:
Missing line at end of file. I suggest setting "Insert Final Newline" and "Trim Final Newlines" in vscode to do it automatically

Reply:
done